modules/pam_pwhistory/opasswd.c

```diff
@@ -139,7 +139,7 @@ compare_password(const char *newpass, const char *oldpass)
   outval = crypt (newpass, oldpass);
 #endif
 
-  retval = outval != NULL && strcmp(outval, oldpass) == 0;
+  retval = outval != NULL && pam_consttime_streq(outval, oldpass);
```

Comment:
I'm not sure pam_pwhistory needs this treatment, ...

Reply:
Dropped.